finetuner/tuner/augmentation.py

```diff
@@ -55,6 +55,8 @@ def _vision_preprocessor(
             blob = doc.blob
         else:
             raise AttributeError('Can not load `blob` field from the given document.')
```

Comment:
I just saw this in the previous PR. Can you maybe show the content of the blob field to the user or somehow show the user which case (url or blob) failed?